TWLight/applications/models.py

```diff
@@ -319,16 +318,24 @@ def update_app_status_on_save(sender, instance, **kwargs):
             instance.date_closed = localtime(now()).date()
             instance.days_open = 0
 
-# Authorize editor to access resource after an application is saved as approved
-# or sent. We need to use post_revision_commit from django-reversion to get the
-# **current** revision user. Just listening to the post_save signal will grab
-# the user from the previously saved version.
+# Authorize editor to access resource after an application is saved as sent.
 @receiver(post_save, sender=Application)
 def post_revision_commit(sender, instance, **kwargs):
-    if reversion.is_active() and (instance.status == 2 or instance.status == 4) and not instance.imported:
+    # Check if an authorization already exists.
+    if instance.specific_stream:
+        existing_authorization = Authorization.objects.filter(
+            authorized_user=instance.user,
+            partner=instance.partner,
+            stream=instance.specific_stream).exists()
+    else:
+        existing_authorization = Authorization.objects.filter(
+            authorized_user=instance.user,
+            partner=instance.partner).exists()
+
+    if instance.status == Application.SENT and not instance.imported: #This work is so close to being done. Final hurdle is that revisions don't get saved with coordinator user when marking sent anymore.
```

Comment:
this looks way better than what I was doing. Is the comment on this line still relevant?

Reply:
Oh, nope - this is outdated, I fixed it on line 338, switching from using `reversion` to going direct to `sent_by` which we already have at this point.

Reply:
In fact it looks like I remembered to remove it in https://github.com/WikipediaLibrary/TWLight/pull/246/commits/9dc9c9e6bd1027d23d365793c411c8a36206a952 :D

Reply:
works for me!